skyportal/models/tag.py

```diff
@@ -14,5 +12,22 @@ class ObjTagOption(Base):
     tag_name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
 
 
-ObjTags = join_model("obj_tags", ObjTagOption, Source)
+ObjTags = join_model("obj_tags", ObjTagOption, Obj)
```

Comment:
please drop the `s`. As you noticed the table names do not have an `s` in their name. This is because the python name is lowercased and appended an `s` when autogenerating the postgres table names.